R/parameters.R

```diff
@@ -164,21 +170,33 @@
 #' clinically diseased (these values refer to the index in drug_* parameters); default = NULL, NULL, NULL
 #' * clinical_treatment_coverage - a vector of coverage values for each drug; default = NULL, NULL, NULL
 #'
-#' PEV parameters: 
+#' MDA, SMC and PMC parameters:
+#' please set these parameters with the convenience functions
+#' `set_mda`, `set_smc` and `set_pmc`,
+#' with `peak_season_offset`
+#'
+#' bednet, irs and mosquito feeding cycle parameters:
+#' please set vector control strategies using `set_betnets` and `set_spraying`
+#'
+#' * bednets - boolean for if bednets are enabled; default = FALSE
+#' * phi_bednets - proportion of bites taken in bed; default = 0.85
+#' * k0 - proportion of females bloodfed with no net; default = 0.699
+#' * spraying - boolean for if indoor spraying is enabled; default = FALSE
+#' * phi_indoors - proportion of bites taken indoors; default = 0.90
+#'
+#'
+#' PEV parameters:
 #' please set vaccine strategies with the convenience functions in
-#' `pev_parameters.R:set_pev_epi`
-#' `pev_parameters.R:set_mass_pev`
+#' `pev_parameters.R::set_pev_epi`
```

Comment:
Is this a thing?

If we're going to make these references more formal shouldn't we use Roxygen [syntax](https://stuff.mit.edu/afs/athena/software/r/current/RStudio/resources/roxygen_help.html)?

```suggestion
#' \code{\link{set_pev_epi}}
```

Reply:
Good shout! I'm not super familiar with the syntax, so thanks for pointing this out. I've now changed all the references to other functions to links to the help pages.